gridencoder/grid.py

```diff
@@ -178,7 +178,7 @@ def grad_total_variation(self, weight=1e-7, inputs=None, bound=1, B=1000000):
         S = np.log2(self.per_level_scale) # resolution multiplier at each level, apply log2 for later CUDA exp2f
         H = self.base_resolution # base resolution
 
-        if inputs is None:
+        if inputs is None inputs.size(0) == 0:
```

Comment:
missing a `or`?

Reply:
Yes, I did mean `if inputs is None or inputs.size(0) == 0:`

Reply:
but the code is missing an `or`, maybe you could commit again to fix it?